build.gradle

```diff
@@ -62,6 +63,7 @@ dependencies {
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     }
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
When did we decide to upgrade to JUnit 4 from JUnit 5?

Reply:
Corrected